uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -24,21 +25,16 @@ define([
     $log.debug('Component: manage-leave-requests');
 
     var vm = this;
-    var actionMatrix = {};
     var filterByAll = { name: 'all', label: 'All' };
 
-    actionMatrix[sharedSettings.statusNames.awaitingApproval] = ['respond', 'cancel', 'approve', 'reject'];
-    actionMatrix[sharedSettings.statusNames.moreInformationRequired] = ['edit', 'cancel'];
-    actionMatrix[sharedSettings.statusNames.approved] = ['edit'];
-    actionMatrix[sharedSettings.statusNames.cancelled] = ['edit'];
-    actionMatrix[sharedSettings.statusNames.rejected] = ['edit'];
-
     vm.absencePeriods = [];
     vm.absenceTypes = [];
+    vm.absenceTypesIndexed = {}; // This object is needed for leaveRequestActionDropdown component
```

Comment:
Done. The problem was that staff-leave-report uses indexed object and manage-leave-report uses array, so this has to be considered.